src/Symfony/Component/Yaml/Tests/YamlTest.php

```diff
@@ -24,6 +25,56 @@ public function testParseAndDump()
         $this->assertEquals($data, $parsed);
     }
 
+    public function testParseWithMultilineQuotes()
+    {
+        $yaml = <<<YAML
+foo:
+  bar: 'baz
+biz
+
+'
+  baz: 'Lorem
+
+       ipsum'
+  error: Une erreur s'est produite.
+  trialMode: 'période d''essai'
+  double_line: 'Les utilisateurs sélectionnés
+n''ont pas d''email.
+
+'
+  a: 'b''
+c'
+  empty: ''
+  foo:bar: 'foobar'
+YAML;
+
+        $this->assertSame(['foo' => [
+            'bar' => "baz biz\n",
```

Comment:
```suggestion
            'bar' => 'baz biz ',
```

Reply:
Is this the behavior you expect @xabbuh? Currently with this input we get this result without your suggestion.

Reply:
Yes, as far as I know that should be the outcome.

Reply:
In the YAML spec I didn't find sth. for this case so it might be a very special case. This is the ChatGPT answer for that:
> An empty line is treated as a line break (``\n``)

This is also the way it works in different online interpreters which are supporting this (like this one: http://www.yaml-online-parser.appspot.com/.
This one also interpretes
```
baz: 'Lorem

   ipsum'
```
As ``{"baz": "Lorem\nipsum"}``.
While other interpreters like https://codebeautify.org/yaml-parser-online interpret this as ``{"baz": "Lorem ipsum"}``.

So it seems to be not really clear how to deal with this and we should decide how to deal with it. My suggestion would be "keep it simple for this very special case" and keep it how it is atm in this PR with the ``\n``.  What do you think?